enhancements/storage/csi-azure-patches.md

```diff
@@ -0,0 +1,217 @@
+---
+title: cloud-provider-azure-dependency-update-in-azure-csi-drivers
+authors:
+  - "@fbertina"
+reviewers:
+  - "@openshift/storage"
+  - "@JoelSpeed"
+approvers:
+  - "@openshift/storage"
+  - "@JoelSpeed"
+api-approvers:
+  - None
+creation-date: 2024-02-26
+last-updated: 2024-02-26
+tracking-link:
+  - https://issues.redhat.com/browse/STOR-1764
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# cloud-provider-azure dependency update for Azure CSI drivers
+
+## Summary
+
+Azure cloud logic from embedded in Azure CSI drivers rely on the
+`cloud-provider-azure` dependency.  As a result of that, many bug
```

Comment:
I think the full name `sigs.k8s.io/cloud-provider-azure` should be used at least on few first occurrences.